notty-community.opam

```diff
@@ -1,17 +1,19 @@
-# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
 synopsis: "Declaring terminals"
-description:
-  "Notty is a declarative terminal library for OCaml structured around a notion of composable images. It tries to abstract away the basic terminal programming model, providing something simpler and more expressive."
+description: """
+Notty is a declarative terminal library for OCaml structured around a notion
+of composable images. It tries to abstract away the basic terminal
+programming model, providing something simpler and more expressive.
+"""
 maintainer: [
   "Ali Caglayan <alizter@gmail.com>"
   "Antonin Décimo <antonin@tarides.com>"
   "Romain Calascibetta <romain.calascibetta@gmail.com>"
```

Comment:
I guess this needs to be updated? Or are these people still available to maintain this package?

Reply:
For my part, I'm available :+1: